src/cdk/coercion/string-array.spec.ts

```diff
@@ -0,0 +1,28 @@
+import {coerceStringArray} from '@angular/cdk/coercion/string-array';
+
+describe('coerceStringArray', () => {
```

Comment:
Looks like there also needs to be a test with a custom delimiter

Reply:
The `trim values` test uses a custom delimiter: https://github.com/angular/components/blob/c3747bde041ba73566e833a888e91b93aa0571b2/src/cdk/coercion/string-array.spec.ts#L14

Is it better to add another test to explicitly test this behavior?

Reply:
Ah, I missed that (lots of commas there). It would be nice to have an explicit test for it